pkg/kv/kvserver/testdata/replica_raft_overload/min_live_match_index

```diff
@@ -0,0 +1,19 @@
+# Start out in a situation in which s3 is overloaded. The quorum (1,2) is ahead
+# of the min live index, so we can afford pausing replication to s3.
+run voters=(1@100,2@80,3@60) overloaded=(3) min-live-match-index=60
```

Comment:
We should have additional tests that an overloaded follower that's below min-live-match-index will get cut off. The comment in the code seems to imply that we almost didn't do this, but I think we definitely want this behavior since an overloaded follower is rather likely to be behind on replication already.